examples/Makefile

```diff
@@ -33,6 +33,8 @@ ifeq (${OS},Darwin)
 shared_lib = lib${1}.dylib
 else ifeq (${OS},Linux)
 shared_lib = lib${1}.so
+else ifeq (${OS},OpenBSD)
```

Comment:
I love that you are adding FreeBSD support, but I'd rather that be split out into a separate PR.